### ISSUE TITLE
Problem about performance

### ISSUE THREAD

**Issue by BrianWayland** [ChiWeiHsiao/DeepVO_PyTorch#36]
As is shown in the result, the performance of trained model is very unsatisfying. Actually, after 100 epochs valid loss and train loss is so close, simply improving number of epoch can do nothing to performance. My problem is are there any solutions to promote the performance of the model?
![route_03_gradient](https://user-images.githubusercontent.com/32663715/99534094-d867e500-29e1-11eb-9a0f-e269a7f9e6ad.png)
![route_04_gradient](https://user-images.githubusercontent.com/32663715/99534100-da31a880-29e1-11eb-82a0-01be7741b865.png)
![route_05_gradient](https://user-images.githubusercontent.com/32663715/99534104-dbfb6c00-29e1-11eb-9a30-f15bbc5a3037.png)
![route_06_gradient](https://user-images.githubusercontent.com/32663715/99534107-dd2c9900-29e1-11eb-8a10-4e71de2e3aed.png)
![route_07_gradient](https://user-images.githubusercontent.com/32663715/99534108-ddc52f80-29e1-11eb-989c-b0a70a7edef1.png)
![loss](https://user-images.githubusercontent.com/32663715/99533993-b1a9ae80-29e1-11eb-9cbe-b36f004c85dc.png)

**Comment by knsjoon** [comment]
@BrianWayland  My valid loss is also very low but when I test them, I get very bad results as well. 
Have you solved this issue?

**Comment by quanqhow** [comment]
Hello, I also got the same bad result as shown below, so can anyone solve this problem?
![route_04](https://user-images.githubusercontent.com/16968752/105795309-a0899a80-5f5a-11eb-92cc-52d53a3196fe.png)
![route_05](https://user-images.githubusercontent.com/16968752/105795344-ab442f80-5f5a-11eb-9179-ab97d2fecc32.png)
![route_07](https://user-images.githubusercontent.com/16968752/105795345-abdcc600-5f5a-11eb-800c-c2497337c4df.png)
![route_09](https://user-images.githubusercontent.com/16968752/105795346-abdcc600-5f5a-11eb-8b90-0266a9ef807c.png)
![route_10](https://user-images.githubusercontent.com/16968752/105795347-abdcc600-5f5a-11eb-9631-3f932c47e5a4.png)

**Comment by mariakalimeri** [comment]
I was not even able to reproduce these results. I trained a model with the same data and the loss definition and hyperparameters in the current commit, which seem to be the ones used to train the for the provided model in the README, but my results after 250 epochs are way worst than this.
